src/OpenTK.Windowing/NativeWindow.cs

```diff
@@ -0,0 +1,216 @@
+﻿//
+// GameWindow.cs
```

Comment:
I'm afraid you're lying to me.

Reply:
Updated. I really hate this practice probably because I don't understand the reasoning behind it.

Reply:
On retrospective now that I'm not copy-pasting, it seems fairly insane to include the name in two places.